apps/wallet-mobile/src/components/Analytics/Analytics.tsx

```diff
@@ -0,0 +1,206 @@
+import React from 'react'
+import {defineMessages, useIntl} from 'react-intl'
+import {Image, ScrollView, StyleSheet, Switch, View} from 'react-native'
+
+import image from '../../assets/img/analytics.png'
+import {Button, Text} from '../../components'
+import {useMetrics} from '../../metrics/metricsManager'
+import {spacing} from '../../theme'
+
+type Props = {
+  type: 'notice' | 'settings'
+  onClose?: () => null
+}
+
+export const Analytics = ({type, onClose}: Props) => {
+  const intl = useIntl()
```

Comment:
here a example how we add the strings

https://github.com/Emurgo/yoroi/blob/develop/apps/wallet-mobile/src/features/Menu/Menu.tsx#L187